test/react/functional-components/equivalence.js

```diff
@@ -0,0 +1,87 @@
+var React = require('React');
+// the JSX transform converts to React, so we need to add it back in
+this['React'] = React;
+
+function joinClasses(className) {
```

Comment:
Can we cut this stuff out of the test? It seems like the essence is just having two equivalent elements.